### PR TITLE
[fix bug 1413992] Add Wait Face campaign download page

### PR DESCRIPTION
**Pull request by alexgibson**
## Description
- Adds a new "Wait Face" campaign download template at `/firefox/new/?xv=waitface`.
- Page is localized for `en-US` & `de` only.
- New base template is heavily borrowed from the fx-lifestyle layout, but the design made enough changes to warrent spending the time to refactor and convert to Pebbles.

## Issue / Bugzilla link
https://bugzilla.mozilla.org/show_bug.cgi?id=1413992

## Testing
Demo: https://bedrock-demo-agibson.us-west.moz.works/firefox/new/?xv=waitface

- Test in all major browsers, plus IE > 9.

### CODE REVIEW

**Comment by flodolo** [comment]
We need an add_lang_file in all 3 templates. I'd suggest

```
{% add_lang_files "firefox/new/wait-face" %}
```

**Comment by alexgibson** [comment]
Ok, this should be good for code review.

**Comment by alexgibson** [comment]
This has [been approved](https://bugzilla.mozilla.org/show_bug.cgi?id=1413992#c15) and can merge once reviewed.

File: bedrock/firefox/templates/firefox/new/wait-face/scene1.html [hunk 1]
```diff
@@ -0,0 +1,26 @@
+{# This Source Code Form is subject to the terms of the Mozilla Public
+ # License, v. 2.0. If a copy of the MPL was not distributed with this
+ # file, You can obtain one at http://mozilla.org/MPL/2.0/. -#}
+
+{% add_lang_files "firefox/new/wait-face" %}
+
+{% extends "firefox/new/wait-face/base.html" %}
+
+{% block page_css %}
+  {% stylesheet 'firefox_new_wait_face' %}
```
Comment (jpetto) [inline bedrock/firefox/templates/firefox/new/wait-face/scene1.html]: Could this be in the base template?

File: media/css/firefox/new/wait-face.scss [hunk 2]
```diff
@@ -0,0 +1,264 @@
...
+        text-align: left;
+    }
+}
+
+/* -------------------------------------------------------------------------- */
+// Top hero section
+
+.intro {
+    background: #0a0d0f url('/media/img/firefox/new/wait-face/purple-wave.png') bottom left no-repeat;
+    min-height: 0;
```
Comment (jpetto) [inline media/css/firefox/new/wait-face.scss]: Is this needed?

File: bedrock/firefox/templates/firefox/new/wait-face/base.html [hunk 3]
```diff
@@ -0,0 +1,105 @@
...
+{% block page_desc %}{{_('Millions of people around the world trust Firefox Web browsers on Android, iOS and desktop computers. Fast. Private. Download now!')}}{% endblock %}
+
+{% block page_css %}{% endblock %}
+
+{% block body_id %}firefox{% endblock %}
+{% block body_class %}{% endblock %}
+
+{% block string_data %}
+  data-pixels="{% for pixel in settings.TRACKING_PIXELS %}{{ pixel }}{% if not loop.last %}::{% endif %}{% endfor %}"
+{% endblock %}
```
Comment (jpetto) [inline bedrock/firefox/templates/firefox/new/wait-face/base.html]: Can we move this into the scene 2 template?

File: bedrock/firefox/templates/firefox/new/wait-face/base.html [hunk 4]
```diff
@@ -0,0 +1,105 @@
...
+          </p>
+        {% endblock %}
+        </li>
+        <li>
+        {% block content_3 %}
+          <h3>{{ _('Stands for good') }}</h3>
+          <p>
+          {% trans %}
+            Did you know Firefox works with the not-for-profit Mozilla? We keep the Internet healthier by supporting
+            programs like tech education for girls, truth in online journalism, and promoting civility in comments
```
Comment (jpetto) [inline bedrock/firefox/templates/firefox/new/wait-face/base.html]: We need to change "tech education for girls" to "tech education for women". @peiying2 is aware of this change, and will comment here on how we should handle it in this template.

(FWIW, the German translation does not appear to need updating, as it reads "Für die digitale Bildung von Frauen." - "For the digital education of women.")

**Comment by peiying2** [comment]
@jpetto go ahead and make the change. We will update the en-US side.

**Comment by flodolo** [comment]
> @jpetto go ahead and make the change. I will update the en-US version on bedrock-l10n.

I've already updated the file. 

P.S. not sure why you mention bedrock-l10n here. Curiously, the file wasn't pushed to prod yet when I checked last night.

**Comment by flodolo** [comment]
> I've already updated the file.

We'll need to check the demo server to verify that everything remains translated.

**Comment by alexgibson** [comment]
@jpetto thanks for the review - I updated and pushed to demo

**Comment by flodolo** [comment]
> We'll need to check the demo server to verify that everything remains translated.

Confirmed it's working.

**Pull request merged**